manager/cve_handler.py

```diff
@@ -261,7 +261,7 @@ def handle_patch(cls, **kwargs):
                 return cls.format_exception('At least one given CVE must exist', 404)
             (CveAccountData.insert_many(to_insert, fields=cls._fields)
              .on_conflict(conflict_target=cls._conflict_target, preserve=[], update=values).execute())
-        except (IntegrityError, psycopg2IntegrityError):
+        except (IntegrityError, psycopg2IntegrityError, DataError):
```

Comment:
does it also apply for business_risk values? they are implemented same way

Reply:
yes, I updated it